app/controllers/documents_controller.rb

```diff
@@ -15,4 +15,8 @@ def destroy
       redirect_to overview_documents_path
     end
   end
+
+  def current_path(params = {})
```

Comment:
why is this being overwritten? i thought current_path was a rails thing

Reply:
I don't know. Questions controller has an explicit method called current_path, so I added one here. I know that questions controller is set up to use the question path as a param, though, so I should check and make sure this isn't extraneous.